source/blender/gpu/GPU_material.h

```diff
@@ -178,6 +178,7 @@ typedef enum GPUDynamicType {
 	GPU_DYNAMIC_LAMP_COEFFCONST      = 13 | GPU_DYNAMIC_GROUP_LAMP,
 	GPU_DYNAMIC_LAMP_COEFFLIN        = 14 | GPU_DYNAMIC_GROUP_LAMP,
 	GPU_DYNAMIC_LAMP_COEFFQUAD       = 15 | GPU_DYNAMIC_GROUP_LAMP,
+	GPU_DYNAMIC_LAMP_DYNAREAMAT      = 16 | GPU_DYNAMIC_GROUP_LAMP,
```

Comment:
Can you move this line before under GPU_DYNAMIC_LAMP_DYNSPOTSCALE to order DYN and non DYN uniform (DYN means that the uniform can't be make constant).